Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gemspec
 
 # Private runtime dependencies
+gem 'nokogiri', '~> 1.13', force_ruby_platform: true
```

Comment:
I'm not sure that you can use force_ruby_platform in the Gemfile itself. I believe you need to put it into the Bundler config file

Reply:
You can... as long as your bundler is new enough